components/Team.tsx

```diff
@@ -0,0 +1,41 @@
+import team from 'data/team.json';
+import Image from 'next/image';
+
+export const Team = () => (
+  <section id="team" className="bg-[#cddecc]">
+    <div className="max-w-screen-2xl mx-auto">
+      <h1 className="pt-10 text-center max-sm:text-5xl">Team</h1>
+      <div className="grid grid-cols-1 md:grid-cols-3 overflow-auto flex-wrap gap-x-8 xl:gap-x-16 px-12">
+        {team.map((member, index) => {
+          let finalClass = '';
+
+          if (team.length % 2 !== 0 && index == team.length - 1) {
+            finalClass = 'md:row-span-1';
```

Comment:
unsure this is needed, can you clarify? on the current website if there's only one item in the last row, it'll occupy the first cell in a 2-column, this happen automatically without providing this class

Reply:
I think this was a leftover from having a flex-col before and then realising that grid was better 😓 Will remove!